VogtPayroll5/Employee.cs

```diff
@@ -27,13 +27,25 @@ public decimal CalculateGrossPay(int hours, decimal rate)
             }
         }
 
+        public decimal CalculateOvertimePay(int hours, decimal rate)
+        {
+            int baseHours = 40;
+
+            if (hours >= 40)
```

Comment:
```
        public int EmpID { get; set; }
        public string EmpName { get; set; }
        public int HoursWorked { get; set; }
        public decimal HourlyPayRate { get; set; }
```

Rather than have the setters exposed publically, these should not have a public setter.
```c#
        public int EmpID { get; }
        public string EmpName { get; }
        public int HoursWorked { get; }
        public decimal HourlyPayRate { get; }
```

Better to set these via a constructor so that anyone who uses the Employee class must follow the `contract` defined by the constructor. I.e., they HAVE to provide all 4 parameters to instantiate an employee.